.github/scripts/automergeCrowdinPr.js

```diff
@@ -81,21 +97,20 @@ module.exports = async ({ github, context }) => {
       event: 'APPROVE',
       body: `Approved from [${context.workflow} #${context.runNumber}](${context.serverUrl}/${context.repo.owner}/${context.repo.repo}/actions/runs/${context.runId}).`,
     })
-  } else {
-    console.log(`Already reviewed`)
   }
 
-  const hasAutomergeLabel = pr.labels.some((label) => label.name === AUTOMERGE_LABEL)
-  if (!hasAutomergeLabel) {
-    console.log(`Adding ${AUTOMERGE_LABEL} label`)
-    await github.rest.issues.addLabels({
-      owner,
-      repo,
-      issue_number: pr.number,
-      labels: [AUTOMERGE_LABEL],
-    })
-  } else {
-    console.log(`Already labelled with ${AUTOMERGE_LABEL}`)
-  }
+  console.log(`Bringing PR #${pr.number} up to date with main branch`)
+  await github.rest.pulls.updateBranch({
+    owner,
+    repo,
+    pull_number: pr.number,
+  })
+
+  console.log(`Enabling automerge on PR #${pr.number}`)
+  await github.graphql(enableAutomergeQuery, {
```

Comment:
interestingly / annoyingly, enabling automerge is not supported via the rest api's and only supported with this graphql endpoint. this part is important since we have jobs that are triggered by the creation of branches and we won't be able to successfully call the `merge` rest endpoint since the CI won't have finished when these jobs are run.